crates/turbopack-core/src/issue/mod.rs

```diff
@@ -397,11 +397,37 @@ impl CapturedIssues {
 }
 
 #[turbo_tasks::value]
-#[derive(Clone)]
-pub struct IssueSource {
+#[derive(Clone, Debug)]
+pub struct LazyIssueSource {
     pub source: Vc<Box<dyn Source>>,
-    pub start: SourcePos,
-    pub end: SourcePos,
+    pub start: usize,
+    pub end: usize,
+}
+
+#[turbo_tasks::value_impl]
+impl LazyIssueSource {
+    #[turbo_tasks::function]
+    pub fn new(source: Vc<Box<dyn Source>>, start: usize, end: usize) -> Vc<Self> {
+        Self::cell(LazyIssueSource { source, start, end })
+    }
+
+    #[turbo_tasks::function]
+    pub async fn to_issue_source(self: Vc<Self>) -> Result<Vc<IssueSource>> {
```

Comment:
I couldn't find a straightforward way to use Rust's `From`/`To` with Vcs. Any ideas?